File: method.py
class Bank:
    def __init__(self,name,amount) :
        self.name = name
        self.amount = amount
    def loan(self,tk):
        if tk>self.amount:
            print("You can not take loan more than your money")
        else:
            print("Loan Granted")

padma = Bank('padma',500)
padma.loan(1000)

File: method.cpp
#include<bits/stdc++.h>

using namespace std;

class Bank{
    public:
    string name;
    int  amount;
    //constructor
    Bank(string name,int amount)
    {
        this->name = name;
        this->amount = amount;
    }

    //inside the class definition
    void display_name(){
        cout<<"Thank for banking with "<<name<<endl;
    }

    void loan(int v);
};


//outside method definition
void Bank::loan(int tk){
    if (tk>amount){
        cout<<"You can not take loan more than half of your money";
    }
    else{
        cout << "Loan Granted";
    }
}
                    
int main(){
    Bank Bank1("padma",500);
    Bank1.display_name();
    Bank1.loan(100);     
    return 0;
}